From: Stefan Bader <stefan.bader@canonical.com>
Date: Thu, 08 Aug 2013 10:43:24 +0200
Subject: Fix path to qemu-dm

When creating a configuration we can completely leave out the path
to qemu-dm and the Xen tools will extend it to the default path.
Though that means that the testing is also broken and I am not sure
how that could be fixed. The output of configuration would always be
different and also different between Xen releases as those binaries
are in per-release version paths. Up to a certain point alternates
where used but that is gone.

Signed-off-by: Stefan Bader <stefan.bader@canonical.com>
Updated: 2014-02-17

Index: virt-manager-1.3.2/virtinst/guest.py
===================================================================
--- virt-manager-1.3.2.orig/virtinst/guest.py	2015-12-24 22:17:03.526012208 +0100
+++ virt-manager-1.3.2/virtinst/guest.py	2015-12-24 22:17:03.522012119 +0100
@@ -816,10 +816,7 @@
             return
 
         if self.os.is_hvm() and self.type == "xen":
-            if self.conn.caps.host.cpu.arch == "x86_64":
-                self.emulator = "/usr/lib64/xen/bin/qemu-dm"
-            else:
-                self.emulator = "/usr/lib/xen/bin/qemu-dm"
+            self.emulator = "qemu-dm"
 
     def _set_cpu_defaults(self):
         self.cpu.set_topology_defaults(self.vcpus)
